specs/SelectMulti.spec.ts

```diff
@@ -1,66 +1,136 @@
 import { mount, Wrapper } from '@vue/test-utils'
 
+import { SelectOption } from '../src/types'
+
 import SelectMulti from '../src/SelectMulti.vue'
+import { 
+	simpleOptions,
+	optionsWithCustomLabelAndUniqueIdFields
+} from './sampleData'
 
 describe('SelectMulti', () => {
 	let wrapper: Wrapper<Vue>
+	let options: SelectOption[]
 
 	describe('given simple options', () => {
-		beforeEach(() => {
-			// TODO: Share sample data between spec files & Storybook stories
-			const options = [
-				{ label: 'Item One', value: 'one' },
-				{ label: 'Item Two', value: 'two' },
-				{ label: 'Item Three', value: 'three' }
-			]
+		beforeEach(() => options = simpleOptions )
 
-			wrapper = mount({
-				data() { 
-					return { selectedOptions: [], options }},
-				template: '<div><SelectMulti :options="options" v-model="selectedOptions" label="Sample SelectMulti" /></div>',
-				components: { SelectMulti }
+		describe('and a simple SelectMulti initialized with options, v-model, and a label', () => {
```

Comment:
And by "simple" we mean "does not have custom labels or unique id fields beyond the value property" correct? Just want to make sure i understand the nomenclature

Reply:
Yeah, I wonder if I can word it better. What I mean is "these are the bare minimum props for the component to work at all", so it's actually not specific to missing `labelField` or `uniqueIdField`. I'm gonna do:

`and a SelectMulti initialized with the bare minimum props: options, v-model, and a label`

if that sounds good?